composer.json

```diff
@@ -28,8 +28,7 @@
     "require": {
         "php": "^7.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "slevomat/coding-standard": "dev-master#469368cc6d8fe83aceba259ef65f1c0a2f63055b",
-        "squizlabs/php_codesniffer": "^3.5.0"
+        "slevomat/coding-standard": "dev-master#db3b3b50edef49cdbe88547536e4f3562eed6737"
```

Comment:
Why removing the phpcs dependency?

Reply:
my bad, probably should stay? but slevomat/coding-standard implicitly requires 3.5.3 so perhaps no need for it here 🤔 WDYT?

Reply:
See https://github.com/slevomat/coding-standard/blob/db3b3b50edef49cdbe88547536e4f3562eed6737/composer.json#L14: it's ok for us to use this as a transitive dependency since we don't directly depend on phpcs.

Reply:
AFAIK it was there to get rid of bugs in phpcs' earlier versions but the 3.5.3 boundary is high enough.

Reply:
Yes, there is no own sniff in this repository currently so no need depend on phpcs.

Reply:
> it's ok for us to use this as a transitive dependency since we don't directly depend on phpcs

We do, there're a lot of rules in `ruleset.xml` that comes from PHPCS :)

Reply:
@carusogabriel yeah, you are right :) my bad